migrations/20200514193400-remove-fed-add-ca-departments.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+var _ = require('lodash');
+var yaml = require('js-yaml');
+var fs = require('fs');
+
+var dbm;
+var type;
+var seed;
+
+/**
+  * We receive the dbmigrate dependency from dbmigrate initially.
+  * This enables us to not have to rely on NODE_PATH.
+  */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = function(db, callback) {
+  db.runSql(`
+    delete from tagentity where type = 'agency' 
+  `, callback);
+
+  var agencyFile = 'tools/init-tag-data/agency.yml';
+  if (fs.existsSync(agencyFile)) {
+    var agencyFileObjects = yaml.safeLoad(fs.readFileSync(agencyFile).toString());
+    agencyFileObjects.forEach(agency => {
+	  db.insert('agency', [
```

Comment:
`migrations/20200429000000-remove-fed-add-ca.js` adds `CDPH` & `DSS`. This results in duplicate entries for those.

The delete statement above should remove from the `agency` table instead of the `tagEntity` table.

Reply:
D'oh. Sloppy. Nice catch.